tests/design_assistant/processing/test_design_assistant_persistence.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from slamd.design_assistant.processing import design_assistant_persistence
+
+
```

Comment:
Tests failen, bitte an anderen Persistence Tests orientieren

Reply:
Vorschlag: Erstmal den Test einfach löschen. Ich mach das dann einfach in der Validierungsstory